tests/internal_heating_statistics/screen-output

```diff
@@ -24,25 +24,25 @@ Number of degrees of freedom: 13,764 (8,450+1,089+4,225)
    Solving Stokes system... 0 iterations.
 
    Postprocessing:
-     Temperature min/avg/max:                1000 K, 1001 K, 1001 K
-     Internal heating rate (average/total):  1 W/kg, 3.4e+15 W
+     Temperature min/avg/max:       1000 K, 1001 K, 1001 K
+     Heating rate (average/total):  3400 W/kg, 1.156e+19 W
```

Comment:
Before, the output was just the specific internal heating rate. However, the term we use in the assembly is density * specific_heating_rate, and that is what the output of the postprocessor is now. It's not backward compatible, but I didn't see an easy way to change this, and all of the other postprocessors write the heating rate (W/m³) instead of the specific heating rate (W/kg). 

But I still have to update that in the statistics postprocessor (it still has the wrong unit), and we should definitely document that somewhere!